### PR TITLE
Prepare release of scanner rules add-ons

### PR DESCRIPTION
**Pull request by thc202**
- Active scanner rules;
 - Active scanner rules (beta);
 - Active scanner rules (alpha);
 - Advanced SQLInjection Scanner;
 - DOM XSS Active scanner rule;
 - Passive scanner rules;
 - Passive scanner rules (beta);
 - Passive scanner rules (alpha).

### CODE REVIEW

File: addOns/ascanrules/CHANGELOG.md [hunk 1]
```diff
@@ -186,3 +186,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 
 
+[33]: https://github.com/zaproxy/zap-extensions/releases/ascanrules-v33
```
Comment (kingthorin) [inline addOns/ascanrules/CHANGELOG.md]: The asterisk at L50 seems to be making everything below italicized. Replace with html entity?

Comment (kingthorin) [reply addOns/ascanrules/CHANGELOG.md]: &#42;nix (`&#42;`)

Comment (thc202) [reply addOns/ascanrules/CHANGELOG.md]: Actual rendering looks fine: https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrules/CHANGELOG.md

**Comment by thc202** [comment]
Need to update the about.html page of DOM XSS.

File: addOns/ascanrules/CHANGELOG.md (hunk 1 above)
Comment (kingthorin) [reply addOns/ascanrules/CHANGELOG.md]: Huh, ok just a diff view issue

**Comment by thc202** [comment]
Help page updated.

**Pull request merged**